brainiak/event_seg/event.py

```diff
@@ -0,0 +1,312 @@
+"""Event segmentation using a Hidden Markov Model"""
+
+# Authors: Chris Baldassano and Cătălin Iordan (Princeton University)
+
+import numpy as np
+from scipy import stats
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "EventSegment",
+]
+
+
+class EventSegment:
+    """Class for learning and representing an event segmentation of
+    continuous fMRI data
+
+    Attributes
+    ----------
+    n_events: int
+        Number of segments to learn
+
+    step_var: lambda : default lambda step: 4 * (0.98 ** (step - 1))
+        The Gaussian variance to use during fitting, as a function of the
+        number of steps. Should decrease slowly over time.
+
+    max_steps: int : default 500
+        Maximum number of steps to run during fitting
+
+
+    p_start, p_end: row vectors
+        initial and final prior distributions over events
+
+    P: array
```

Comment:
Please add the shape for all arrays. Also, you should write `numpy.ndarray` or simply `ndarray` (unless you are using `array.array`).